package.json

```diff
@@ -1,100 +1,36 @@
 {
   "name": "ember-autofocus-modifier",
-  "version": "5.0.0",
-  "description": "The default blueprint for ember-cli addons.",
-  "keywords": [
-    "ember-addon"
-  ],
-  "publishConfig": {
-    "registry": "https://registry.npmjs.org/"
-  },
+  "version": "0.0.0",
```

Comment:
Question: so this doesn't affect the version of the package since it's located in `addon/package.json` right?

Reply:
correct